wodles/gcloud/tools.py

```diff
@@ -93,6 +123,63 @@ def get_file_logger(output_file: str, level: int = 3) -> logging.Logger:
 
     return logger_file
 
+
 def get_wazuh_queue() -> str:
-    """Get Wazuh queue"""
-    return os.path.join(common.find_wazuh_path(), 'queue', 'sockets', 'queue')
+    """Get Wazuh queue
+
+    Returns
+    -------
+    A str containing the path to Wazuh queue"""
+    return os.path.join(find_wazuh_path(), 'queue', 'sockets', 'queue')
+
+
+@lru_cache(maxsize=None)
+def find_wazuh_path() -> str:
+    """
+    Gets the path where Wazuh is installed dynamically
+
+    Returns
+    -------
+    A str path where Wazuh is installed or empty string if there is no framework in the environment
+    """
+    abs_path = os.path.abspath(os.path.dirname(__file__))
+    allparts = []
+    while 1:
+        parts = os.path.split(abs_path)
+        if parts[0] == abs_path:  # sentinel for absolute paths
+            allparts.insert(0, parts[0])
+            break
+        elif parts[1] == abs_path:  # sentinel for relative paths
+            allparts.insert(0, parts[1])
+            break
+        else:
+            abs_path = parts[0]
+            allparts.insert(0, parts[1])
+
+    wazuh_path = ''
+    try:
+        for i in range(0, allparts.index('wodles')):
+            wazuh_path = os.path.join(wazuh_path, allparts[i])
```

Comment:
```suggestion
        wazuh_path = os.path.join(wazuh_path, *[allparts[i] for i in range(allparts.index('wodles'))])
```

An elegant one-liner for that part would be this.